packages/beep/package.json

```diff
@@ -39,7 +39,7 @@
     "rollup": "^1.20.0"
   },
   "devDependencies": {
-    "rollup": "^1.20.0",
+    "rollup": "^2.0.0-2",
```

Comment:
Isn't it prerelease version?

Reply:
Ah, I see note. Sorry

Reply:
Actually, I could update it with the real release now

Reply:
Updated